tests/baseclass.py

```diff
@@ -46,6 +47,25 @@ def setUp(self):
         self._device = _ped.device_get(self.path)
         self.device = parted.getDevice(self.path)
 
+# Base class for any test case, which requires loop block device - need run
```

Comment:
I see you follow the convention in the rest of this file but the convention is kind of wrong. I'd prefer if we star having these comments as doc strings instead.